utils/constants.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+export const BASE_API_PATH = "/api/ism";
```

Comment:
why is this "/api/ism" vs backend path is "/_opendistro/ism"?

Reply:
Kibana/nodejs convention is to prefix with `api`, we could add `_opendistro` if we want to further namespace it (`/api/_opendistro/ism`), but it follows the current convention with alerting.

Reply:
ok lets keep the convention then.